tasks/task1130_xcsr_vi_commonsense_mc_classification.json

```diff
@@ -0,0 +1,6014 @@
+{
+    "Contributors": [
+        "Maitreya Patel"
+    ],
+    "Source": [
+        "X-CSR-QA dataset",
+        "Link: https://inklab.usc.edu/XCSR/xcsr_datasets"
+    ],
+    "Categories": [
+        "Multiple-Choice Question"
+    ],
+    "Input_language": "vi",
+    "Output_language": "en",
+    "Definition": "In this task, you will be presented with a multiple-choice question, and you should answer the question based on your commonsense knowledge. Moreover, the question and options are given in Vietnamese language. Classify the answers based on the given options.",
```

Comment:
Say what is the expected output (is it the answer string or "A", "B", "C", ...)

Reply:
Shorten the instructions. I think you can merge and shorten the first two sentences.

Reply:
(obviously, these apply to all tasks)

Reply:
I have updated all definitions!!